extensions/cdc-postgres/src/main/java/com/hazelcast/jet/cdc/postgres/PostgresCdcSources.java

```diff
@@ -123,7 +127,39 @@ private Builder(@Nonnull String name) {
             config.setProperty(CdcSourceP.SEQUENCE_EXTRACTOR_CLASS_PROPERTY, PostgresSequenceExtractor.class.getName());
             config.setProperty(ChangeRecordCdcSourceP.DB_SPECIFIC_EXTRA_FIELDS_PROPERTY, "schema");
             config.setProperty("database.server.name", UuidUtil.newUnsecureUuidString());
-            config.setProperty("snapshot.mode", "exported");
+            config.setProperty("snapshot.mode", "initial");
+        }
+
+        /**
+         * Snapshot mode that will be used by the connector.
+         *
+         * If you want to use {@link io.debezium.connector.postgresql.PostgresConnectorConfig.SnapshotMode#CUSTOM},
+         * please use {@link #setCustomSnapshotter(Class)} method instead.
+         */
+        @Nonnull
+        public Builder setSnapshotMode(@Nonnull PostgresSnapshotMode snapshotMode) {
+            PostgresConnectorConfig.SnapshotMode debeziumMode;
+            switch (snapshotMode) {
```

Comment:
I would prefer to stick with the IntelliJ default formatting to avoid whitespace changes when others will work on this piece of the code

Reply:
Good point, done. I was too much inspired by new switch expressions ;P